packages/component-submission/server/resolvers/submitManuscript.js

```diff
@@ -51,7 +56,8 @@ async function submitManuscript(_, { data }, { user, ip }) {
     Manuscript.statuses.MECA_EXPORT_PENDING,
   )
 
-  await mecaExport(manuscript, S3Storage.getContent, ip)
+  // This function can take a while so do not await this (apart from in tests)
+  return mecaExport(manuscript, S3Storage.getContent, ip)
```

Comment:
It is important that the mecaExport is done OOB so that the response to show the Thank You page to the user is displayed straight away.
/cc @hdrury1 @chugginselifesciences